docs/user-guide/commands/argocd_relogin.md

```diff
@@ -15,7 +15,21 @@ argocd relogin [flags]
 ### Examples
 
 ```
-argocd relogin --password YOUR_PASSWORD
+  
+# Reinitiates the login with previous contexts
+argocd relogin
+
+# Reinitiates the logins with sso
+argocd relogin --sso 
```

Comment:
The `--sso` flag doesn't exist for the `relogin` command

Reply:
Thanks for correcting @crenshaw-dev 
updated the PR